mne/viz/evoked.py

```diff
@@ -15,9 +15,11 @@
 from numbers import Integral
 
 import numpy as np
+import matplotlib.lines as mlines
```

Comment:
it probably doesn't matter for the backend choosing because you didn't import `pyplot`, but let's nest this one, too, to keep `mne` import times down

Reply:
Ah I see, you told me to put it at the top of the *function*, not the file. So that's what I'll do.